airflow/bin/cli.py

```diff
@@ -69,7 +69,6 @@
 from airflow.www_rbac.app import create_app as create_app_rbac
 from airflow.www_rbac.app import cached_appbuilder
 
-from sqlalchemy import func
```

Comment:
the commit of this line is 1 year ago(https://github.com/apache/incubator-airflow/commit/b9a87a07e3b6dbd965c91f09d0f1c10281f628f9).

Reply:
are we sure this line is not using?

Reply:
is it because we remove the func usage in some commits recently?

Reply:
Exactly. The func usage was removed in https://github.com/apache/incubator-airflow/commit/53e707428826d603f43aebb73e18fe077881c4b3#diff-1c2404a3a60f829127232842250ff406 which was merged three days ago.